com.unity.netcode.gameobjects/TestHelpers/Runtime/NetcodeIntegrationTest.cs

```diff
@@ -111,6 +111,32 @@ public enum HostOrServer
 
         private NetworkManagerInstatiationMode m_NetworkManagerInstatiationMode;
 
+        private bool m_EnableVerboseDebug;
+
+        /// <summary>
+        /// Used to display the various integration test
+        /// stages and can be used to log verbose information
+        /// for troubleshooting an integration test.
+        /// </summary>
+        /// <param name="msg"></param>
+        protected void VerboseDebug(string msg)
```

Comment:
it'd be nice to have aggressiveinline attribute on top of this method.

Reply:
also nit: perhaps `VerboseDebugLog`?

Reply:
I agree.